src/Configurator/DockerComposeConfigurator.php

```diff
@@ -128,6 +132,73 @@ public function unconfigure(Recipe $recipe, $config, Lock $lock)
         $this->write('Docker Compose definitions have been modified. Please run "docker-compose up" again to apply the changes.');
     }
 
+    public static function shouldConfigureDockerRecipe(Composer $composer, IOInterface $io, Recipe $recipe): bool
+    {
+        if (null !== self::$configureDockerRecipes) {
+            return self::$configureDockerRecipes;
+        }
+
+        $dockerPreference = $composer->getPackage()->getExtra()['symfony']['docker'] ?? null;
+        if (null !== $dockerPreference) {
+            self::$configureDockerRecipes = $dockerPreference;
+
+            return self::$configureDockerRecipes;
+        }
+
+        if ('install' !== $recipe->getJob()) {
+            // default to not configuring
+            return false;
+        }
+
+        $warning = $io->isInteractive() ? 'WARNING' : 'IGNORING';
+        $io->writeError(sprintf('  - <warning> %s </> %s', $warning, $recipe->getFormattedOrigin()));
+        $question = '    The recipe for this package contains some Docker configuration.
+
+    This may create/update <comment>docker-compose.yml</comment> or update <comment>Dockerfile</comment> (if it exists).
+
+    Do you want to include Docker configuration from recipes?
+    [<comment>y</>] Yes
+    [<comment>n</>] No
+    [<comment>p</>] Yes permanently, never ask again for this project
+    [<comment>x</>] No permanently, never ask again for this project
+    (defaults to <comment>y</>): ';
+        $answer = $io->askAndValidate(
+            $question,
+            function ($value) {
+                if (null === $value) {
+                    return 'y';
+                }
+                $value = strtolower($value[0]);
+                if (!\in_array($value, ['y', 'n', 'p', 'x'])) {
+                    throw new \InvalidArgumentException('Invalid choice.');
+                }
+
+                return $value;
+            },
+            null,
+            'y'
+        );
+        if ('n' === $answer) {
+            self::$configureDockerRecipes = false;
+
+            return self::$configureDockerRecipes;
+        }
+        if ('y' === $answer) {
+            self::$configureDockerRecipes = true;
+
+            return self::$configureDockerRecipes;
+        }
+
+        // yes or no permanently
+        self::$configureDockerRecipes = 'p' === $answer ? true : false;
```

Comment:
`= 'p' === $answer;`

Reply:
My editor was suggesting the same thing I now realize :)